modules/ebi-metagenomics/assessmcpproportions/main.nf

```diff
@@ -89,6 +89,8 @@ process ASSESSMCPPROPORTIONS {
     """
     touch ${assess_mcp_prop_prefix}_mcp_cons.tsv
 
+    echo 'dummy' > ${assess_mcp_prop_prefix}_mcp_cons.tsv
```

Comment:
that's fine but is there a reason you prefer to keep it non-empty?

Reply:
nf-core liniting don't like empty files (which overall I think it's good)
Having stubs with some content is useful IMO

Reply:
BTW, this is not a good example.. a real piece of data on this file would be useful